### PR TITLE
Suggestions

### PR DESCRIPTION
**Pull request by Dheeraj-Thikkamaneni**
find suggestions and show top five suggestions in UI

### CODE REVIEW

**Review by shirishma1 — CHANGES_REQUESTED** [review]
can we adjust the spacing of extra lines in the code

**Comment by Dheeraj-Thikkamaneni** [comment]
> can we adjust the spacing of extra lines in the code

**Comment by Dheeraj-Thikkamaneni** [comment]
> can we adjust the spacing of extra lines in the code

yeah, sure will do, its just my habit for my personnel ease for readability

File: spellcheck.py [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
 import ast
-from flask import Flask, render_template, request
+from flask import Flask, jsonify, render_template, request
 
 
 def read_file():
     words = []
-    for i in range(1, 627):
-        with open('for_irish\irish_list_'+str(i)+'.txt', 'r', encoding='utf-8') as file:
+    for i in range(1, 15):
```
Comment (gantlavamshikrishnareddy) [inline spellcheck.py:9]: why have you used less amount of data for English than Irish? would you think the frequency list would be good for the data given to training for English and contain all words?

**Pull request merged**